src/tags/htmltag.js

```diff
@@ -240,4 +240,11 @@ function toAttribute(key, value) {
   }
 }
 
+function escapeQuotes(obj) {
+    return Object.keys(obj).map(key => utils.isString(obj[key]) ?
```

Comment:
The function accepts an object of key=>value but returns an array of value[]. You need to return an object with key=>value.

Reply:
done - reverted back to forEach